docs/modules/ROOT/pages/type-definitions/schema-configuration/index.adoc

```diff
@@ -0,0 +1,35 @@
+[[type-definitions-schema-configuration]]
+= Schema Configuration
+
+Neo4j GraphQL Library supports several functionalities such as CRUD operations, aggregation, filtering, and others.
+To make them work, a large amount of GraphQL types are generated.
+
+For instance, starting from the type definitions:
+
+[source, graphql, indent=0]
+----
+type Movie { 
+    title: String
+    length: Int
+} 
+----
+
+In some cases, it may be advisable to reduce the scope of the API produced. Whatever the reason is to reduce the size of the schema, improve the performance, or limit access to unwanted operations, this section provides all the information needed to configure the API produced.
+
+From the type definitions above, let's say we want to prevent users from querying `movies` in the Neo4j Database. To do that, let's bring some changes to the initial type definitions:
```

Comment:
```suggestion
Consider the previously mentioned type definitions.
If you want to prevent users from querying `movies` in the Neo4j Database, you need to make the following the changes:
```